src/main/webapp/app/course/learning-paths/components/learning-path-lecture-unit/learning-path-lecture-unit.component.ts

```diff
@@ -29,6 +31,11 @@ export class LearningPathLectureUnitComponent {
     readonly isLoading: WritableSignal<boolean> = signal(false);
     readonly lectureUnit = signal<LectureUnit | undefined>(undefined);
 
+    readonly lecture = computed(() => this.lectureUnit()?.lecture);
+
+    readonly isCommunicationEnabled = computed(() => isCommunicationEnabled(this.lecture()?.course));
+    readonly isMessagingEnabled = computed(() => isMessagingEnabled(this.lecture()?.course));
```

Comment:
Looking at the implementation of `isCommunicationEnabled` and `isMessagingEnabled`, it looks to me like `isCommunicationEnabled` is enough for this view:

```
export function isCommunicationEnabled(course: Course | undefined) {
    const config = course?.courseInformationSharingConfiguration;
    return config === CourseInformationSharingConfiguration.COMMUNICATION_AND_MESSAGING || config === CourseInformationSharingConfiguration.COMMUNICATION_ONLY;
}

export function isMessagingEnabled(course: Course | undefined) {
    const config = course?.courseInformationSharingConfiguration;
    return config === CourseInformationSharingConfiguration.COMMUNICATION_AND_MESSAGING;
}
```

i.e., if `isMessagingEnabled` is true, then `isCommunicationEnabled` is also true.

```suggestion
    readonly isCommunicationEnabled = computed(() => isCommunicationEnabled(this.lecture()?.course));
```